### DISCUSSION TITLE
How do you change the superset name on the favicon

### DISCUSSION THREAD

**Discussion by esambu** [apache/superset#13209] [Q&A / Help]
Am able to change the favicon logo but the name superset is still there...how can i go around it changing the name?

**Comment by Vijay-LionOrbit** [comment]
@esambu I need to change the superset favicon logo. Please tell me how were able to do it.

**Comment by santosh-sahoo334** [comment]
You need to Create a customized docker image by copying the favicon image to the container path. 
```
COPY my_dir/my_favicon.png /app/superset/static/assets/images/
## Change the config.py as below
FAVICONS = [{"href": "/static/assets/images/my_favicon.png"}]
# Copy config.py with modified image locations.
COPY config.py /app/superset/
```
